Apromore-Custom-Plugins/Calendar-Portal/src/main/resources/static/calendar/zul/calendars.zul

```diff
@@ -42,28 +42,28 @@
       </hbox>
       <hbox sclass="ap-window-header-actions" hflex="1" align="start" pack="start" spacing="0"
             style="padding: 8px;">
-        <button id="addNewCalendarBtn" iconSclass="z-icon-plus-circle" tooltiptext="Add a new calendar">
-          Add new calendar
+        <button id="addNewCalendarBtn" iconSclass="z-icon-plus-circle" tooltiptext="${$composer.labels.add_calendar_text}">
+          ${$composer.labels.add_calendar_text}
         </button>
       </hbox>
       <hbox hflex="1" align="start" pack="start" spacing="0"
             style="padding:10px 0px 5px 10px;">
-        <label value="Available custom calendars:"/>
+        <label value="${$composer.labels.available_calendars_text}"/>
       </hbox>
       <listbox multiple="false" sclass="ap-calendars-listbox" id="calendarListbox" vflex="1" hflex="1" nonselectableTags="input">
         <listhead>
-          <listheader align="left" label="Used" width="48px"/>
-          <listheader align="left" label="Name" hflex="1" />
-          <listheader align="left" label="Created on" width="120px" />
-          <listheader align="center" label="Actions" width="108px"/><!-- icons -->
+          <listheader align="left" label="${$composer.labels.applied_text}" width="48px"/>
```

Comment:
This label has changed from "Used" to "Applied". The width of the listheader is currently too small for the text "Applied". We should either increase the width or change the text to something smaller.